packages/realm/package.json

```diff
@@ -68,9 +68,16 @@
     "build:ios:simulator": "wireit",
     "build:ios:ios": "wireit",
     "build:ios:catalyst": "wireit",
-    "install": "prebuild-install --runtime napi || echo 'Failed to download prebuild for Realm'"
+    "install": "prebuild-install --runtime napi || echo 'Failed to download prebuild for Realm'",
+    "docs": "wireit"
   },
   "wireit": {
+    "docs": {
+      "command": "typedoc",
+      "dependencies": [
+        "bundle"
+      ]
+    },
```

Comment:
A nit, but the other properties of this object is in the same order as the scripts above. I'd move this entry to the bottom of the `wireit` object.